### PR TITLE
Fixed the DevUtils tests

### PR DESCRIPTION
**Pull request by jalextowle**
## Description

Fixes the DevUtils tests and removes the `.only` modifier.

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: contracts/integrations/test/dev-utils/get_order_hash.ts [hunk 1]
```diff
@@ -35,11 +44,11 @@ blockchainTests('DevUtils.getOrderHash', env => {
...
             takerAssetAmount: new BigNumber(0),
             expirationTimeSeconds: new BigNumber(0),
-            exchangeAddress,
+            exchangeAddress: exchange.address,
             chainId,
         };
-        expect(await devUtils.getOrderHash(order, new BigNumber(chainId), exchangeAddress).callAsync()).to.be.equal(
-            expectedOrderHash,
+        expect(await devUtils.getOrderHash(order, new BigNumber(chainId), exchange.address).callAsync()).to.be.equal(
+            orderHashUtils.getOrderHashHex(order),
```
Comment (dorothy-zbornak) [inline contracts/integrations/test/dev-utils/get_order_hash.ts]: :sunglasses:

File: contracts/integrations/test/dev-utils/lib_asset_data.ts [hunk 2]
```diff
@@ -63,7 +62,7 @@ const KNOWN_STATIC_CALL_ENCODING = {
 };
 
 // TODO(jalextowle): This file could really be cleaned up by using the DeploymentManager tool.
-blockchainTests('LibAssetData', env => {
+blockchainTests.resets('LibAssetData', env => {
```
Comment (dorothy-zbornak) [inline contracts/integrations/test/dev-utils/lib_asset_data.ts]: Nice catch.

File: contracts/integrations/test/dev-utils/lib_asset_data.ts [hunk 3]
```diff
@@ -423,8 +422,8 @@ blockchainTests('LibAssetData', env => {
 
         it('should return a balance of MAX_UINT256 if the the StaticCallProxy assetData contains data for a successful staticcall', async () => {
             const staticCallData = staticCallTarget.isOddNumber(new BigNumber(1)).getABIEncodedTransactionData();
-            const trueAsBuffer = ethUtil.toBuffer('0x0000000000000000000000000000000000000000000000000000000000000001');
-            const expectedResultHash = ethUtil.bufferToHex(ethUtil.sha3(trueAsBuffer));
+            const trueAsString = '0x0000000000000000000000000000000000000000000000000000000000000001';
+            const expectedResultHash = hexUtils.hash(trueAsString);
```
Comment (dorothy-zbornak) [inline contracts/integrations/test/dev-utils/lib_asset_data.ts]: Nit:
```suggestion
            const expectedResultHash = hexUtils.hash(hexUtils.leftPad(1));
```

File: contracts/integrations/test/dev-utils/lib_asset_data.ts [hunk 4]
```diff
@@ -434,8 +433,8 @@ blockchainTests('LibAssetData', env => {
 
         it('should return a balance of 0 if the the StaticCallProxy assetData contains data for an unsuccessful staticcall', async () => {
             const staticCallData = staticCallTarget.isOddNumber(new BigNumber(0)).getABIEncodedTransactionData();
-            const trueAsBuffer = ethUtil.toBuffer('0x0000000000000000000000000000000000000000000000000000000000000001');
-            const expectedResultHash = ethUtil.bufferToHex(ethUtil.sha3(trueAsBuffer));
+            const trueAsString = '0x0000000000000000000000000000000000000000000000000000000000000001';
+            const expectedResultHash = hexUtils.hash(trueAsString);
```
Comment (dorothy-zbornak) [inline contracts/integrations/test/dev-utils/lib_asset_data.ts]: Ditto.

File: contracts/integrations/test/dev-utils/lib_asset_data.ts (hunk 3 above)
Comment (jalextowle) [reply contracts/integrations/test/dev-utils/lib_asset_data.ts]: Good call!

**Pull request merged**